tests/Transport/Base.php

```diff
@@ -894,4 +894,17 @@ public function testBodyDataFormat() {
 		$this->assertEquals(httpbin('/post'), $result['url']);
 		$this->assertEquals(array('test' => 'true', 'test2' => 'test'), $result['form']);
 	}
+
+	public function test303GETmethod() {
+		if ($this->skip_https) {
+			$this->markTestSkipped('SSL support is not available.');
+			return;
+		}
+
+		$data    = array('test' => 'true', 'test2' => 'test');
+		$request = Requests::post(httpbin('/status/303', true), array(), $data, $this->getOptions(array('follow_redirects' => true)));
+
+		$this->assertSame(200, $request->status_code);
+		$this->assertsame(httpbin('/get'), $request->url);
```

Comment:
```suggestion
		$this->assertSame(httpbin('/get'), $request->url);
```

Reply:
Fixed, but as function names are case-insensitive, it's not the reason the test keeps failing.